### PR TITLE
Fix typo and aggregation button consistency 

### PR DESCRIPTION
**Pull request by bikramkawan**
Closes : #398

### CODE REVIEW

File: src/column/NumberColumn.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class NumberColumn extends AVectorColumn<number, INumericalVector
   constructor(data: INumericalVector, orientation: EOrientation, $parent: d3.Selection<any>, matrixCol?: MatrixColumn) {
     super(data, orientation);
     this.$node = this.build($parent);
-    this.attachListener();
+    //  this.attachListener();
```
Comment (sgratzl) [inline src/column/NumberColumn.ts]: why not listening to events anymore?

Comment (bikramkawan) [inline src/column/NumberColumn.ts]: This event was responsible creating new matrix view from the converted vector only.So to make consistent as described in issue. I turned off the listener.But we can restore whole matrix from the conversion icon available on the matrix header.

**Comment by NMO13** [comment]
I decided to remove the attachListener() method completely because I think that it won't be used in the future again.

File: src/column/NumberColumn.ts [hunk 2]
```diff
@@ -86,11 +85,12 @@ export default class NumberColumn extends AVectorColumn<number, INumericalVector
...
   }
 
-  setFixedWidth(width:number) {
-    if(isNaN(width)) {
+  setFixedWidth(width: number) {
+    if (isNaN(width)) {
       return;
     }
     super.setFixedWidth(width);
+    super.setFixedWidth(width);
```
Comment (sgratzl) [inline src/column/NumberColumn.ts]: why called twice?

Comment (NMO13) [inline src/column/NumberColumn.ts]: Looks like an automerge misbehaviour.

**Comment by NMO13** [comment]
I have noticed that this PR introduces https://github.com/Caleydo/taggle/issues/412 again.

**Comment by sgratzl** [comment]
> I have noticed that this PR introduces #412 again.

are you sure that this branch is uptodate with the latest master branch?

**Pull request merged**